_posts/2019-10-17-how-to-achieve-cis-benchmark-compliance.adoc

```diff
@@ -0,0 +1,723 @@
+walkthrough] for comprehensive documentation on how `infrastructure-live`, `infrastructure-modules`, and the core IaC
+modules interact.
+ +
+[[benefits]]
+=== Benefits
+This modular, decomposed approach allows for maximum code reuse. The core modules can be used with or without
+compliance, depending on how they are configured. The compliance wrappers are like shadows of the core
+modules; they pass through most of the variables to the core modules without alteration, but hard code any
+settings needed for compliance. When you call the compliance modules from your own code in
+`infrastructure-modules`, you only need to set up any variables that are custom for your environment. Often
+times the default settings are good enough.
+ +
+
+If you need to brush up on how the IaC Library works, read the
+link:https://gruntwork.io/guides/foundations/how-to-use-gruntwork-infrastructure-as-code-library/[How to use
+the Gruntwork Infrastructure as Code Library] guide.
+
+[[deployment_walkthrough]]
+== Deployment walkthrough
+This section is a step-by-step walkthrough for achieving compliance with the AWS Foundations Benchmark
+out-of-the-box using the Infrastructure as Code modules from Gruntwork.
+
+[[pre_requisites]]
+=== Pre-requisites
+
+This walkthrough has the following pre-requistes:
+
+Gruntwork Infrastructure as Code Library::
+  This guide uses code from the https://gruntwork.io/infrastructure-as-code-library/[Gruntwork Infrastructure as Code Library], as it
+  implements most of the production-grade design for you out of the box. Make sure to read
+  link:/guides/foundations/how-to-use-gruntwork-infrastructure-as-code-library[How to use the Gruntwork Infrastructure as Code Library].
+
+Gruntwork Compliance for CIS AWS Foundations Benchmark::
+  This guide also uses code from the https://gruntwork.io/compliance/cis-aws-foundations-benchmark/[Gruntwork CIS AWS
+  Foundations Benchmark repository], which contains the necessary configurations to achieve compliance.
++
+IMPORTANT: You must be a [js-subscribe-cta]#Gruntwork Compliance subscriber# to access the Gruntwork
+Infrastructure as Code Library and the CIS AWS Foundations Benchmark modules.
+
+Terraform::
+  This guide uses https://www.terraform.io/[Terraform] to define and manage all the infrastructure as code. If
+  you're not familiar with Terraform, check out
+  https://blog.gruntwork.io/a-comprehensive-guide-to-terraform-b3d32832baca[A Comprehensive Guide to Terraform],
+  https://training.gruntwork.io/p/terraform[A Crash Course on Terraform], and
+  link:/guides/foundations/how-to-use-gruntwork-infrastructure-as-code-library[How to Use the Gruntwork
+  Infrastructure as Code Library].
+
+[[manual_steps]]
+=== Manual steps
+A few of the Benchmark recommendations are not achievable via API and require a one-time manual configuration.
+Perform the steps in this section manually.
+
+[[root_mfa]]
+==== Enable MFA for the root account
```

Comment:
This is part of https://gruntwork.io/guides/foundations/how-to-configure-production-grade-aws-account-structure/

Reply:
Made reference to this.